catroid/src/org/catrobat/catroid/ui/dialogs/SignInDialog.java

```diff
@@ -470,7 +471,7 @@ public void onGoogleExchangeCodeComplete() {
 
 	@Override
 	public void onGetFacebookUserInfoTaskComplete(String id, String name, String locale, String email) {
-		Log.d("FB", "User Info complete");
+		Log.d(TAG, "User Info complete");
```

Comment:
to not loose information here it would be better to include the "FB" in the message: "FB User Info complete"